main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
 what the date and time of the show is, which row the seat is in, and where in the row the 
 seat is (and likely other factors as well). You can't just give them any ticket in your supply,
-as they are not interchangeable (and may even have different prices). See also [fungible](#fungible).
+as they are not interchangeable (and may have different prices). See also [fungible](#fungible).
 
 ## Notifier
 
-You can track updates to contract state using a notifier. The notifier provides a
-stream of updates describing changes to the state of an offer.
-
-See more: [Notifier](/distributed-programming.md)
+A notifier provides a stream of updates describing changes to the state of an [offer](#offer).
+For more information, see the [Notifier section in the Distributed JavaScript Programming Guide](/distributed-programming.md#notifiers).
 
 ## Object Capabilities
 
-Objects have state, behavior, and references. Lets say Object A has references to Objects B and C, while B and C do not have references to each other. Thus, A can communicate with B and C, and B and C cannot communicate with each other.
-There is an effective zero-cost firewall between B and C.
+Objects have state, behavior, and references. Lets say Object A has references to Objects B 
+and C, while B and C do not have references to each other. Thus, A can communicate with B and C, 
+and B and C cannot communicate with each other. There is an effective zero-cost firewall between B and C.
 
-An *object capability system* constrains how references are obtained. You can't get one just by knowing the name of a global variable or a public class. You can get a reference in only three ways. 
+An *object capability system* constrains how references are obtained. You can't get one just by 
+knowing the name of a global variable or a public class. You can only get a reference via: 
 - Creation: Functions that create objects get a reference to them.
 - Construction: Constructors can endow their constructed objects with references, including inherited references. 
 - Introduction: 
   - A has references to B and C. 
-  - B and C  do *not* have references to each other
+  - B and C do *not* have references to each other
   - A sends B a reference to C. 
     - B now has a reference to C and can communicate with C. 
 
-If references can only be obtained by creation, construction, or introduction, you may have a safe system. If they can be obtained in any other way, your system is unsafe.
+If references can only be obtained by creation, construction, or introduction, you may have a safe 
+system. If they can be obtained in any other way, your system is unsafe.
 
 For more information, see [Douglas Crockford on Object Capabilities](https://frontendmasters.com/courses/good-parts-javascript-web/object-capabilities/).
 
+## Offer
+
+Offers are a structured way of describing user intent. In Zoe, an offer consists of a [proposal](#proposal) (the 
```

Comment:
I think I started the "describing user intent" thing, but it's too vague for a glossary and not definitional enough. 
```suggestion
Users interact with contract instances by making offers. In Zoe, an offer consists of a [proposal](#proposal) (the 
```